neural_compressor/common/base_config.py

```diff
@@ -337,10 +378,10 @@ def from_dict(cls, config_dict: OrderedDict[str, Dict], config_registry: Dict[st
         assert len(config_dict) >= 1, "The config dict must include at least one configuration."
         num_configs = len(config_dict)
         name, value = next(iter(config_dict.items()))
-        config = config_registry[name].from_dict(value)
+        config = config_registry[name]["cls"].from_dict(value)
```

Comment:
How about adding a method to `ConfigRegistry` to get the `cls`, like `get_cls_by_name(name: str) -> BaseConfig`?